kde-apps/svgpart/svgpart-5.9999.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+EGIT_BRANCH="frameworks"
+inherit kde5
+
+DESCRIPTION="Svgpart is a kpart for viewing SVGs"
```

Comment:
We should not repeat the pkg name in  description, maybe "Kpart for viewing SVGs" should be enough.